README.txt

```diff
@@ -1,5 +1,49 @@
 ProActive Agent is a system daemon to automatically starts ProActive runtimes according to a weekly schedule
 
+# Testing
```

Comment:
Maybe it would be better for users to show this part as "Examples" instead of "Testing"
However I like a lot that you add this part.